backend/src/ml_space_lambda/data_access_objects/app_configuration.py

```diff
@@ -205,21 +205,21 @@ def __init__(self, cluster_sizes: list, auto_scaling: EMRAutoScaling, applicatio
 
     def to_dict(self) -> dict:
         return {
-            "clusterSizes": ClusterSize.to_dict(self.cluster_sizes),
+            "clusterTypes": ClusterType.to_dict(self.cluster_sizes),
             "autoScaling": self.auto_scaling.to_dict(),
             "applications": EMRApplications.to_dict(self.applications),
         }
 
     @staticmethod
     def from_dict(dict_object: dict) -> EMRConfig:
         return EMRConfig(
-            ClusterSize.extract_cluster_sizes(dict_object["clusterSizes"]),
+            ClusterType.extract_cluster_sizes(dict_object["clusterTypes"]),
```

Comment:
Do we still want to leave it was `extract_cluster_sizes` if we're using the term types now?

Reply:
good point, will change that